doc/nrf/applications.rst

```diff
@@ -23,3 +23,4 @@ You can also run them on different hardware like a generic development kit, but
    ../../applications/machine_learning/README
    ../../applications/matter_weather_station/README
    ../../applications/zigbee_weather_station/README
+   ../../applications/matter_bridge/README
```

Comment:
Should be between connectivity bridge and asset track, so that they are in alphabetical order based on the page title.
![image](https://github.com/nrfconnect/sdk-nrf/assets/83582261/d108be06-ff9d-431e-be0a-3704fe396f4d)